frontend/src/app/page.tsx

```diff
@@ -1,7 +1,28 @@
+import LandingPageNav from "./components/LandingPage/LandingPageNav"
+import NewWorldText from "./components/LandingPage/HeadingTexts"
+import Paragraphs from "./components/LandingPage/Paragraphs"
+
 export default function Home() {
```

Comment:
nit: maybe rename the file to HomePage or LandingPage instead of just Page?

Reply:
I tested it. in NEXT 13 it doesn't allow us to change the page.tsx to anything else